source/documentation/introduction_to_the_vm_portal/topics/Graphical_User_Interface_elements.adoc

```diff
@@ -8,23 +8,50 @@ image::VM_screen.png[VM Portal screen]
 
 *Key graphical user interface elements*
 
-* image:1.png[title="Header bar"] `Header bar`
+* `Header bar`
 +
-The header bar contains the *Refresh* button, the btn:[User] drop-down button, and the btn:[Messages] drop-down button.
-+
-** The *Refresh* button refreshes the display.
-** The btn:[User] drop-down button displays the following list:
-
-*** *Options*: SSH key, for connecting via serial console to the VM Portal
-*** *About*: VM Portal release information
-*** *Log out*: To log out of the VM Portal
+The header bar contains the following buttons:
 
-** The btn:[Messages] drop-down button displays messages from the system.
-
-* image:2.png[title="Toolbar"] `Toolbar`
+** btn:[Refresh] - manually refreshes the display. See also Refresh Interval.
+** btn:[Account Settings] -  enables setting of logged-in user preferences that are persisted on the server.
+*** btn:[Reset settings] - allows you to reset all options back to initial installation defaults.
+*** General - displays the user name, email address, and allows you to change the UI language.
++
+[NOTE]
+====
+Changing the language setting will override the language chosen in a landing page or by a URL query. See also Advanced Options.
+====
+
+*** Refresh Interval - set the automatic UI refresh rate interval.
+*** Notifications - allows you to mute all notifications for a given time interval. This option is not server persisted, so a page reload will clear this settings.
+*** Console - settings are applied globally for all VMs.
+.. select your preferred console. This will be the first console option  displayed in the VM card (VNC console,  VNC console (Browser),  SPICE console, Remote desktop).
+.. Connect automatically - automatic console login option. Enables automatically connecting to the VM's console of the chosen VM after the user logs in to the VM Portal.
```

Comment:
Log in is detected based on the session age. The default is 60 second  and can be changed using `sessionAgeFirstLoginThresholdInSeconds` parameter in `ovirt-web-ui.config` file. Setting this parameter to zero disables log in detection - console connection will be made on every page reload.

The first console is chosen from the list of types visible in the VM card.